scripts/build.js

```diff
@@ -20,6 +20,7 @@ optParser.addOption('v', 'verbose', 'boolean', 'Whether use verbose level in bui
 optParser.addOption('r', 'rebuild', 'boolean', 'Whether clean before build');
 optParser.addOption('c', 'check', 'boolean', 'Whether check after build');
 optParser.addOption('j', 'jobs', 'string', 'Number of concurrent build jobs');
+optParser.addOption('f', 'features', 'list', 'Specify experinmental features to be enabled. Available features: quic.')
```

Comment:
But '-f quic' has no effect yet?

Reply:
No. I'll remove it. The way to enable QUIC agent is described in source/agent/quic/README.md. Just explicitly build and pack it.

Reply:
Done.